EIPS/eip-5792.md

```diff
@@ -53,10 +55,16 @@ The wallet:
 * MUST send the calls from the address specified in the request's `from` property, if provided
   * If `from` is not provided the wallet SHOULD present the user with an opportunity to view and select the `from` address during confirmation
 * MUST NOT await for any calls to be finalized to complete the batch
-* MUST submit multiple calls as an atomic unit in a single transaction
 * MUST NOT send any calls from the request if the user rejects the request
-* MAY revert all calls if any call fails
-* MUST not execute any further calls after a failed call
+* When atomicity is set to `strict`:
+  * MUST execute all calls atomically in a single transaction.
+* When atomicity is set to `loose`:
+  * MAY execute each call as a single transaction, as part of a bundle that is executed atomically.
+  * MAY execute all calls atomically in a single transaction.
+* When atomicity is set to `none`:
+  * MAY execute each call as a single transaction.
+  * MAY execute each call as a single transaction, as part of a bundle that is executed atomically.
+  * MAY execute all calls sequentially without any atomicity guarantees.
```

Comment:
This doesn't specify the behaviour on failed transactions.

Reply:
Do you think that we need to specify the behaviour for failed transactions submitted sequentially?

I was thinking that we could leave that for the wallets to decide. Meaning that in the simplest form, an app that does not care about atomic execution, will need to go through the receipts and figure out the batch was executed. I would leave further specifications to a different capability (like the [flow-control](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-7867.md) one).